docs/guides/testing.mdx

```diff
@@ -38,6 +38,34 @@ The way you mock a dependency depends on your test runner/library.
 
 In [jest](https://jestjs.io/), you can create a `__mocks__/zustand.js` and place the code in that file. If your app is using `zustand/vanilla` instead of `zustand`, then you'll have to place the above code in `__mocks__/zustand/vanilla.js`.
 
+### TypeScript Usage
```

Comment:
nit: can we change this to sentence case? "TypeScript usage"
(I'm slowly trying to get this consistent across the docs)